external/common/requirements.txt

```diff
@@ -13,9 +13,9 @@ unittest2==0.5.1
 validictory==0.9.1
 PyMySQL==0.6.2
 DBUtils==1.1
-pyproj==1.9.3
+pyproj>=1.9.3
```

Comment:
PyProj 1.9.3 has been replaced on Windows with 1.9.4 Building 1.9.3 from source is problematic. Similar to why a numpy 1.9.1 prebuilt package is used.